cpu/cc2538/clock.c

```diff
@@ -61,8 +61,20 @@
 
 #include <stdint.h>
 /*---------------------------------------------------------------------------*/
-#define RTIMER_CLOCK_TICK_RATIO (RTIMER_SECOND / CLOCK_SECOND)
-#define RELOAD_VALUE (125000 - 1)  /** Fire 128 times / sec */
+#define RTIMER_CLOCK_TICK_RATIO			(RTIMER_SECOND / CLOCK_SECOND)
```

Comment:
Yes, Contiki's whitespace rule is all spaces, no tabs (https://github.com/contiki-os/contiki/blob/master/tools/code-style/uncrustify.cfg).